charts/lunar-proxy/values.yaml

```diff
@@ -23,14 +23,19 @@ env: null
 logLevel: "error"
 tenantName: null
 lunarStreamsEnabled: false
-redisUrl: null
+
 lunarHubURL: null
-redisUseCluster: false
+
 awsAccessKeyId: null
 telemetryEnabled: true
 awsSecretAccessKey: null
 lunarManaged: false
 
+# Redis configuration
+redisUrl: null
+redisUseCluster: false
+redisPrefix: "lunar"
```

Comment:
Can you add the new one to the values schema files?

Reply:
If lunar is the default one, let make it as a constant.